app/views/games/index.html.erb

```diff
@@ -1 +1,165 @@
-<h1>StoicChess</h1>
\ No newline at end of file
+<br />
+<section id="register">
+  <div class="col-sm-5 col-sm-offset-1">
+    <%= image_tag 'chess.jpg', class: 'img-responsive feature_pic' %>
+    <br />
+  </div>
+  <div class="col-sm-3 col-sm-offset-1">
+    <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+        <div class="form-inputs">
+          <br />
```

Comment:
`<br />` isn't really considered good practice IMO. You should be able to use Bootstrap in order to make this happen, but if you really need spacing you could use something like these in CSS (these are my gotos for a base spacing model):
```
.mt32 {
  margin-top: 32px !important;
}
.mt16 {
  margin-top: 16px !important;
}
.mt8 {
  margin-top: 8px !important;
}

.mb32 {
  margin-top: 32px !important;
}
.mb16 {
  margin-bottom: 16px !important;
}
.mb8 {
  margin-bottom: 8px !important;
}
```